ocaml/xapi/xapi_xenops.ml

```diff
@@ -663,9 +663,15 @@ module MD = struct
    * mode, or None otherwise
    *)
   let sriov_vf ~__context vgpu =
+    let is_sriov () =
```

Comment:
Perhaps add a comment saying that this is needed for pass-through of whole-device (which might still be an SRIOV device, but we decide to pass it through in  its entirety instead of individual VF).